FlowCrypt/Functionality/Services/GeneralConstants.swift

```diff
@@ -13,7 +13,7 @@ enum GeneralConstants {
 
     enum Global {
         static let generalError = -1
-        static let attachmentSizeLimit: Int = 10_000_000
+        static let attachmentSizeLimit: Int = 10_485_760
```

Comment:
Either would be fine. One is 10MiB and other is 10MB, neither is more right than the other.

Reply:
Reverted so it more clear